test/pico_time_test/pico_time_test.c

```diff
@@ -364,3 +368,33 @@ int issue_2118_test(void) {
     PICOTEST_END_SECTION();
     return 0;
 }
+
+static int issue_2148_test(void) {
+#if HAS_RP2040_RTC
+    PICOTEST_START_SECTION("Issue #2148 defect - get time after rtc start");
+    struct tm tm = { 0 };
+    struct tm tm_check = { 0 };
+
+    tm.tm_sec = 55;
```

Comment:
I know that this test will probably take _much_ less than 5 seconds to run, but given that the checking-code below is checking for `tm_sec` rollover but **isn't** checking for `tm_min` rollover, perhaps it makes sense to set the `tm_sec` to something like `15` rather than `55` ? :wink:

Reply:
What about 42?